### DISCUSSION TITLE
When building the project with vite the GMapAutocomplete doesn't work

### DISCUSSION THREAD

**Discussion by jeffbruchado** [NathanAP/vue-google-maps-community-fork#55] [Q&A]
This is my code in a default Form:

```
<template>
   <GMapAutocomplete
            v-if="hasAutoComplete"
            @place_changed="setPlace"
            ref="gmapAutoCompleteRef"
          >
            <template #input="slotProps">
              <v-text-field
                color="primary"
                ref="input"
                v-model="newAddress.streetAddress1"
                v-bind="slotProps"
                label="Address Line 1"
                placeholder="Street Address 1"
                :rules="[requiredRule]"
              ></v-text-field>
            </template>
          </GMapAutocomplete>
</template>
```

Running the project locally the autocompletion works fine, but when I build for production, the autocompletion doesn't work.

This is the autocompletion working locally:
![Screenshot 2023-02-20 at 19 07 39](https://user-images.githubusercontent.com/15040760/220205356-1103c7e2-c895-4c97-bd57-3a60126b85e7.png)

When I build the project and access the application in the browser, the autocompletion doesn't work:

![Screenshot 2023-02-20 at 19 08 36](https://user-images.githubusercontent.com/15040760/220205451-f57527db-ac16-4fec-a5fd-2cf6a8cb4d44.png)

In the console I'm receiving the following error:

![Screenshot 2023-02-20 at 19 09 10](https://user-images.githubusercontent.com/15040760/220205511-6574c06e-88d4-43dc-b88f-dd1f1f6e1e6a.png)

Already checked the api keys, and its all loaded correctly, not sure why is this happening.

Someone already encountered a problem like that?

**Comment by NathanAP** [comment]
Hello! Sorry for the delay on my answer.

I'm not sure, but maybe this problem is related to [this line](https://github.com/NathanAP/vue-google-maps-community-fork/blob/main/src/components/autocompleteImpl.js#L37) or [this one](https://github.com/NathanAP/vue-google-maps-community-fork/blob/main/src/components/autocompleteImpl.js#L53). I'm almost sure its related to the first one because both are related to the same variable. 

If thats the case, we need to identify whats causing it ): Do you see something else in your error log?
